extensions/azuremonitor/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "../shared.tsconfig.json",
+	"compilerOptions": {
+		"outDir": "./out",
+		"strict": true,
+		"noUnusedParameters": false,
```

Comment:
Why do you have these two as false? Generally it's good practice to have these enabled and fix any errors that are flagged by it.

Reply:
This was a direct copy from the Kusto extension. I'll update this.

Reply:
I changed these back to false to match the Kusto and Sql extension. Setting these to `true` showed some unused variables in functions that match the other extensions.

Reply:
Please leave them as true and then fix the issues. For the no unused parameters you can replace the parameter name with _ if you truly don't use the parameter (but that should generally not happen, since otherwise why are you passing in a parameter if it's not used?)

Having these enabled encourages good coding practices - all new extensions should be following these.